docs/machine-learning/how-to-guides/load-data-model-builder.md

```diff
@@ -0,0 +1,108 @@
+---
+title: Set up & load data for Model Builder
+description: Learn how to set up and load data from a SQL Server database or a file for use in one of the Model Builder scenarios for ML.NET.
+ms.date: 10/31/2019
+author: luisquintanilla
+ms.author: luquinta
+ms.custom: mvc, how-to
+#Customer intent: As a developer, I want to load data in Model Builder
+---
+
+# Load data in Model Builder
+
+Learn how to set up and load your data sets from a SQL Server or a file database for use in one of the Model Builder scenarios for ML.NET. Model Builder scenarios can use SQL files, image files, and CSV or TSV file formats.
+
+## Data set limitations in Model Builder
+
+Model Builder limits the amount of data you can use for training models:
+
+- Training data set limit: 1GB
+- SQL Server data: 100,000 rows 
+
+[Question: Clarify the limits. Is the SQL Server data limit an absolute limit or for training only? The training dataset  -- is this a limit for text or numerical data from files?]
+
+## Model Builder scenarios 
+
+Model Builder helps you create models for the following machine learning scenarios:
+
+- Sentiment analysis (binary classification): Classify textual data into two categories.
+- Issue classification (multiclass classification): Classify textual data into 3 or more categories.
+- Price prediction (regression): Predict a value from numeric data.
+- Image classification (deep learning): Categorize images based on characteristics.
+- Custom scenario: Build custom scenarios from your data using regression, classification, and other tasks.
+
+This article covers classification and regression scenarios with textual or numerical data, and image classification scenarios. 
+
+## Load text or numeric data from a file  
+
+You can load text or numeric data from a file into Model Builder. It accepts comma-delimited (CSV) or tab-delimited (TSV) file formats. 
+
+1. In the data step of the Model Builder tool, select **File** from the data source dropdown.
+2. Select the button next to the **Select a file** text box, and use File Explorer to browse and select the data file.
+3. Choose a category in the **Column to Predict (Label)** dropdown.
+4. From the **Input Columns (Features)** dropdown, confirm the data columns you want to include are checked.
+
+You're done setting up your data source file for Model Builder. Select the **Train** link to move to the next step in the Model Builder tool.
+
+## Load data from a SQL Server database
+
+Model Builder supports loading data from local and remote SQL Server databases. To use data from SQL Server, you'll first download it to a file. [QUESTION: Is this true? It's confusing -- connect to the database or download??]
```

Comment:
```suggestion
Model Builder supports loading data from local and remote SQL Server databases.
```

You can remove that last line. I agree it's confusing. Internally, it's what Model Builder does, but no need to surface this information.